### ISSUE TITLE
[minor bug] lint: check wasm packages with proper GOOS and GOARCH

### ISSUE THREAD

**Issue by llorllale** [hyperledger/aries-framework-go#1082]
This line in `check_lint.sh`:

`${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} -v $(pwd):/opt/workspace -w /opt/workspace golangci/golangci-lint:v1.21 golangci-lint run`

Runs the linting checks on all packages with the default GOOS and GOARCH, including the new `main` package in cmd/aries-js-worker. There is a silent failure in `make lint` because of this. This is not a big problem because the next line runs the lint checks with the proper GOOS and GOARCH for wasm builds. However, it would be good to remove this extra noise from the build log:

```
~/dev/hyperledger/aries-framework-go$ make lint 
   ... a few lines... 
Running scripts/check_lint.sh
level=warning msg="[runner] Can't run linter unused: buildssa: analysis skipped: errors in package: [/opt/workspace/cmd/aries-js-worker/main.go:12:2: could not import syscall/js (cmd/aries-js-worker/main.go:12:2: build constraints exclude all Go files in /usr/local/go/src/syscall/js)]"
level=warning msg="[runner] Can't run linter goanalysis_metalinter: bodyclose: failed prerequisites: buildssa@github.com/hyperledger/aries-framework-go/cmd/aries-js-worker"
llorllale:~/dev/hyperledger/aries-framework-go$
```

**Comment by troyronda** [comment]
Fixed in #1088

**Comment by llorllale** [comment]
thanks
